src/styles/bar.css

```diff
@@ -5,7 +5,22 @@ body {
   height: 100vh;
 }
 
+
 [role=application] {
   height: initial;
   flex: 1 100vh;
 }
+
+
+[role=banner] {
+  --silver: #d6d8da;
```

Comment:
CSS variables seems to be not very supported ([only in very recent browser](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables)), are they necessary here? /poke GoOz